index.js

```diff
@@ -223,12 +225,20 @@ Storage.prototype.close = function (cb) {
 Storage.prototype.destroy = function (cb) {
   const self = this
   self.close(function () {
-    const tasks = self.files.map(function (file) {
-      return function (cb) {
-        rimraf(file.path, { maxBusyTries: 10 }, cb)
-      }
-    })
-    parallel(tasks, cb)
+    if (self.path) {
+      fs.rm(path.resolve(path.join(self.path, self.name)), { recursive: true, maxRetries: 10 }, function (err) {
```

Comment:
How does this work when saving a torrent into a path that doesn't contain the info hash, like `~/Downloads/my_torrent_name`?

Reply:
ah, i see, it conflicts with single files

Reply:
Or any torrent that isn't saved under an infohash. I have an idea I'll propose in your webtorrent PR.

Reply:
can that even happen? if there is no infohash on torrent [somehow] it will just get a random string https://github.com/feross/fs-chunk-store/pull/17/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R27
I know this is kind of iffy since if the store then isn't deleted, those files are 90% of the time just gonna stay there, but still better than overwriting other existing files

Reply:
I'm talking about for example webtorrent desktop, where the torrent path is set to `~/Downloads/` and a torrent may contain paths like `1.txt`, `aDirectory/2.txt`, etc.

The desired (and current) behavior is that these get saved into `~/Downloads/1.txt`, `~/Downoads/aDirectory/2.txt`, etc.

This is very different from the case I think you're thinking of where each torrent goes into its own directory. In the case I'm describing it's not possible to do reliable cleanup, so we need to somehow disable it in that case.

Reply:
does it not pass the infoHash to the store as `name`?

then that should probably be added separately on wt-d, again, main reason I'm adding this additional directory as an UID is because torrent files were overwriting eachother a lot, causing errors etc etc

Reply:
Some users, like you, want the extra UID to avoid overwriting, but that's just going to be confusing for most users.

At a minimum it should still be possible for webtorrent to keep the current behavior, and in my opinion the current behavior should be the default when `torrentOpts.path` is set.

Adding the infoHash into the path should happen in `webtorrent` so it can be controlled there.